io/net/virt-net/network_virtualization.py

```diff
@@ -841,7 +835,7 @@ def tearDown(self):
         self.session_hmc.quit()
         if 'vios' in str(self.name.name):
             self.session.quit()
-        cmd = "echo 'module ibmvnic -pt; func send_subcrq -pt' > /sys/kernel/debug/dynamic_debug/control"
+        cmd = "echo 'module ibmvnic -p; func send_subcrq -p' > /sys/kernel/debug/dynamic_debug/control"
```

Comment:
same here keep the timestamp
`"echo 'module ibmvnic -pt; func send_subcrq -p' > /sys/kernel/debug/dynamic_debug/control`